ct-app/core/subgraph/entries/safe.py

```
@@ -14,10 +14,10 @@ def __init__(self, address: str, balance: str, allowance: str, owners: list[str]
         :param balance: The balance of the safe.
         :param allowance: The allowance of the safe.
         """
-        self.address = address.lower()
+        self.address = address.lower() if isinstance(address, str) else address
         self.balance = float(balance) if balance else 0
         self.allowance = float(allowance) if allowance else 0
-        self.owners = owners
+        self.owners = [owner.lower() for owner in owner if isinstance(owner, str)] 
```

Ruff (0.8.2)

Undefined name `owner`

(F821)